ballerina/googlesheets_client.bal

```diff
@@ -185,9 +185,15 @@ public isolated client class GoogleSheetsClient {
                 int i = 0;
                 record {} rowArray = {};
                 string[] rowValues = re `,`.split(rowString);
+                if columnNames.length() != self.dataTypes.length() {
+                    return <persist:Error>error("Error: the spreadsheet is not initialised correctly.");
```

Comment:
Shall we say why it is not initialized? `sheets columns do not match with the field types`

Reply:
updated in [7cc2d06](https://github.com/ballerina-platform/module-ballerinax-persist.googlesheets/pull/38/commits/7cc2d06b018923a1e05a4f3cd1f8961834b48bcd)